sickbeard/__init__.py

```diff
@@ -634,7 +634,7 @@ def initialize(consoleLogging=True):
 
         # Check if we need to perform a restore of the cache folder
         restoreDir = os.path.join(DATA_DIR, 'restore')
-        if os.path.exists(restoreDir):
+        if os.path.exists(restoreDir) and os.path.exists(srcDir):
```

Comment:
This wont work as you intend to as the srcDir is define in the function and not accesible from outside.

It should be like this:
```
if os.path.exists(restoreDir) and os.path.exists(os.path.join(restoreDir, 'cache')):
```